tracker/tracker/src/main/app/index.ts

```diff
@@ -274,7 +274,10 @@ export default class App {
       .then(r => {
         if (r.status === 200) {
           return r.json()
-        } else { // TODO: handle canceling && 403
+        } else if (r.status === 403) {
+          warn("OpenReplay tracking has been cancelled");
+          this.stop();
+        } else {
```

Comment:
I'm not sure if this is a good solution, I was unable to launch OpenReplay locally to test it.

I see a couple of issues:
- It should check whether tracking was really canceled. It's possible with `else if (r.status === 403 && r.json().then(json => json.error === "cancel")` but that's very verbose.
- I feel that error and cancel handling should be done in `.catch`
- This `else if` branch does not return anything